### DISCUSSION TITLE
Images stop being loaded and show Internal Server Error instead

### DISCUSSION THREAD

**Discussion by Max2983** [requarks/wiki#6903] [Error / Bug Report]
I run a docker version of Wiki.js. After one or two latest updates I noticed a bug. Now when I upload and insert image into article it looks fine, and shown correctly after publication. But for some unknown reasons after several days those images are not loading and not shown on the article page. When I check the URL path of the image, it is correct. But when i try to open the image URL directly I get 'Internal Server Error'. Please note, that other images I uploaded in the earlier period (before updates) are having the same correct image directory path and loaded and shown on the web-site correctly. I made no changes on server, to the wiki.js core engine or settings whatsoever, because I am not a software or developer by no means. Please, advise on the problem' causes, so I could request a help from ones.

**Comment by HDR** [comment]
We're experiencing the same issue, images will fail to load, and will instead return "Internal Server Error" with"Forbidden" printed in the log, this starts happening after a reboot.

**Reply by Max2983** [reply]
I have updated to the latest version yesterday, and the problem is gone. I don't know if there was any fix in the update.
Several days before this I have upgraded my VPS (added a core and some RAM). I suspect that the problem had place because recent updates does not go well on weak VPS (like 1 CPU core and 8 Gb RAM). Seems like it was not enough to update Wiki.JS properly. I hope this will be fixed anyway.

**Reply by HDR** [reply]
@Max2983 Just updated, the images disappear after the server is rebooted unfortunately

Throws 500 Internal Server Error

![image](https://github.com/requarks/wiki/assets/20230450/cb40420c-9731-4e43-9280-1c3630735847)

![image](https://github.com/requarks/wiki/assets/20230450/f8993255-4485-4390-9d9d-c20f2a24030a)

**Reply by Max2983** [reply]
So sad. I am afraid of rebooting my server now.

**Reply by Max2983** [reply]
Some say it can be a right problem - https://github.com/requarks/wiki/discussions/6957
But I have it allowed to see pictures in my Guest user group rights from the very beginning.
<img width="363" alt="Screenshot 2024-02-01 at 19 39 34" src="https://github.com/requarks/wiki/assets/85618579/89db8746-2af3-47ae-8747-7f9cd185b5ab">

**Reply by HDR** [reply]
yeah, all my users have read permissions for assets too

**Comment by rrg92** [comment]
Some message in log of the server and log of javascript in your webpage?

Try log in in incoginto with same user and another user (admin user) and check if problem happens too. Try check server (output in docker) and browser console

**Comment by Michaelwucoc** [comment]
I am now having the same problem. Now I use external image store service to help me :(

**Comment by Michaelwucoc** [comment]
I think I know the problem..
It seems to be the Local Storge System Error. After disabling that, it will be fine!

**Reply by Max2983** [reply]
I did not try your recipe but noticed several times already that the problem appears and dissolves sporadically. Am not sure but seems like it is connected with wiki.js updates. As if some updates end up causing the problem, and some not. What more important is that sometimes it heals itself automatically.

**Reply by BennyboyH** [reply]
> I think I know the problem.. It seems to be the Local Storge System Error. After disabling that, it will be fine!

Thanks for your input! Disabling it actually solved the problem for me. Tho I still have to look on how to have both working at the same time. But that's a topic for future-me

**Reply by GenericNerd** [reply]
Yep! This worked! I'm not sure why this is, or why it hasn't been solved but it's great to see a working resolution.

**Comment by jcdufourd** [comment]
I had the same problem of images not appearing, and solved it by changing the "Path" of the "Local File System" in the "Storage" section of the admin area from **_relative_** to **_absolute_** (as required in the comment below the field)
